b92.html

```diff
@@ -7,6 +7,22 @@
    <body>
      <div id="top">
       <h2>B92</h2>
+	  <ul>
+	   <li><a href="https://www.youtube.com/watch?v=wiXxifU5ilQ">NOVO</a></li>
```

Comment:
It does not make sense that all buttons are linked to some YT clip. Lets use the real links from B92 site here.